django_project/frontend/src/pages/Admin/Importer/ScheduledJobs/index.jsx

```diff
@@ -128,14 +128,21 @@ export default function ScheduledJobs({ ...props }) {
       },
     })
   ];
+
   return <Fragment>
-    <AdminListContent
+    <AdminList
```

Comment:
You can use this AdminListContent

Reply:
Can check here :
https://github.com/unicef-drp/GeoSight-OS/blob/main/django_project/frontend/src/components/AdminList/Contents/User.jsx